### PR TITLE
Use CLI-based Helm installation for ingress tests

### PR DESCRIPTION
**Pull request by dhawton**
Move `conformance-ingress` workflow to cli-based helm install mode in an effort to complete another item in https://github.com/cilium/cilium/issues/25156

Locally act wasn't cooperating so may need CI to troubleshoot any potential issues.

### CODE REVIEW

File: .github/workflows/conformance-ingress.yaml [hunk 1]
```diff
@@ -95,7 +95,6 @@ jobs:
             --helm-set=ingressController.loadbalancerMode=${{ matrix.loadbalancer-mode }} \
             --helm-set=ingressController.default=${{ matrix.default-ingress-controller }} \
             --helm-set=extraConfig.bpf-lb-acceleration=${{ matrix.bpf-lb-acceleration }} \
-            --rollback=false \
             --version="
```
Comment (sayboras) [inline .github/workflows/conformance-ingress.yaml]: can you remove the version flag as well ? I think this is legacy flag before helm template/install mode is added.

**Review by sayboras — APPROVED** [review]
Thanks and lgtm :heavy_check_mark:

File: operator/pkg/ingress/annotations/annotations.go [hunk 2]
```diff
@@ -33,6 +34,7 @@ const (
 	TCPKeepAliveProbeIntervalAnnotationAlias    = annotation.Prefix + "/tcp-keep-alive-probe-interval"
 	TCPKeepAliveProbeMaxFailuresAnnotationAlias = annotation.Prefix + "/tcp-keep-alive-probe-max-failures"
 	WebsocketEnabledAnnotationAlias             = annotation.Prefix + "/websocket"
+	TLSMinimumProtocolVersionAlias              = annotation.Prefix + "/tls-minimum-protocol-version"
```
Comment (sayboras) [inline operator/pkg/ingress/annotations/annotations.go]: actually the recent push contains some extra changes, is it intentional ? 

I don't think the annotation here is working in term of setting minimum TLS version.

**Review by sayboras — CHANGES_REQUESTED** [review]

File: operator/pkg/ingress/annotations/annotations.go (hunk 2 above)
Comment (dhawton) [reply operator/pkg/ingress/annotations/annotations.go]: Not intentional, sorry.. I was working on something else and thought I had stashed it. Updated commit.

**Review by bimmlerd — APPROVED** [review]

**Review by zacharysarah — APPROVED** [review]
LGTM from a docs perspective

**Review by brlbil — APPROVED** [review]

**Review by sayboras — APPROVED** [review]
LGTM :heavy_check_mark:

**Comment by sayboras** [comment]
The changes are limited to gha Conformance Ingress, which is passed in this PR. All reviews are in, marking this ready to merge.

Thanks a lot for your contribution :medal_military:

**Pull request merged**